.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL = 'http://private-anon-b00bb0b7e8-wbooksapi.apiary-mock.com/api/v1'
```

Comment:
add this file to .gitignogere, We don't usually upload the environment variables to the repo due to security issues